tests/utils/test_integration.py

```diff
@@ -0,0 +1,26 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Integration tests for utils."""
+
+import importlib
+import os
+
+
+def test_all_modules_importable() -> None:
```

Comment:
It looks like you forgot the `@pytest.mark.integration` here.

Reply:
yes, I did. Good catch!

Reply:
Fixed